packages/gatsby/src/schema/types/sort.ts

```diff
@@ -1,84 +1,36 @@
-// @flow
-
-const {
-  getNamedType,
+import {
   getNullableType,
   GraphQLInputObjectType,
   GraphQLList,
-} = require(`graphql`)
-const { addDerivedType } = require(`./derived-types`)
+  GraphQLInputFieldConfigMap,
+} from "graphql"
+import { addDerivedType } from "./derived-types"
+import {
+  InputTypeComposer,
+  SchemaComposer,
+  EnumTypeComposer,
+  InterfaceTypeComposer,
+} from "graphql-compose"
+import { ComposeEnumValueConfig } from "graphql-compose/lib/EnumTypeComposer"
+import { ObjMap } from "graphql-compose/lib/utils/definitions"
```

Comment:
Shouldn't this be available from `graphql-compose` main endpoint - given that `utils/definitions` are re-exported in https://github.com/graphql-compose/graphql-compose/blob/29b237c3f556891fa7cd025500dddeb1cdc12ffd/src/index.d.ts#L37

Reply:
ah, we are on older major :S ok, makes sense, tho I don't like importing internals like that :(

Reply:
Totally agree with you. I'll edit it!